### ISSUE TITLE
When editing axis values, press tab to go directly to the next axis value box

### ISSUE THREAD

**Issue by mrk-z** [googlefonts/fontra#902]
Currently in the right panel, when I finish editing the value of a axis and press tab, the focus first goes to the axis slider, and then the next axis value box.

This is a little redundant, since usually we use mouse to control the sliders, and also if I'm already finishing editing the axis value in the text box, I would not need to switch to the slider to edit it again. Then it also creates extra steps when I want to press tab to go to the next axis.

It would be more intuitive and helpful if we could press tab and go directly to the value box of the next axis.

**Comment by justvanrossum** [comment]
Good point. Especially since we made sure that arrow keys have the same effect whether the numeric input has the focus, or the slider itself.
